spec/webauthn/authenticator_attestation_response_spec.rb

```diff
@@ -53,6 +57,10 @@
       )
     end
 
+    it "verifies" do
+      expect(attestation_response.valid?(original_challenge, original_origin)).to eq(true)
```

Comment:
And also need to use `verify` instead of `valid?` here :-P

Reply:
hahaha the forest behind the tree 🙏